### DISCUSSION TITLE
Querying the ontology

### DISCUSSION THREAD

**Discussion by b3h0ld3r** [ontop/ontop#561] [Q&A] [answered]
Hi!

Is it possible, to query the ontology alongside the virtual graph? I built a test project (Data tables in PostgreSQL - Ontology and mappings in Protege/w Ontop Plugin), and while querying via the Ontop Sparql Tab works fine, the answers only include triplets from the VKG, but triplets from the ontology do not show up. 

Thank you in advance!

Attila

**Comment by apano-on** [comment, marked as answer]
Seems like the same question as #555. Try adding to the properties file:
```
ontop.enableFactExtractionWithTBox=true
```
as per https://ontop-vkg.org/guide/advanced/configuration.html.

**Reply by b3h0ld3r** [reply]
Thank you, that is exactly what I was looking for!
